integration-testing/election-manager/cypress/integration/tallies.ts

```diff
@@ -0,0 +1,33 @@
+import { electionMultiPartyPrimaryWithDataFiles } from '@votingworks/fixtures'
+
+describe('Election Manager can create SEMS tallies', () => {
+  it('Election Manager can tally results properly', () => {
+    cy.visit('/')
+    cy.get('input[type="file"]').attachFile(
+      'electionMultiPartyPrimarySample.json'
+    )
+    cy.contains('Election loading')
+    // cy.contains('Election Hash: abdfbe6a58')
```

Comment:
Uncomment or delete?

Reply:
Woops yeah I changed elections and forgot to update this.